pwa-devdocs/src/product-recs/index.md

```diff
@@ -0,0 +1,88 @@
+Adobe Sensei processes this behavioral data along with the catalog data from the Magento backend and calculates the product associations leveraged by the recommendations service.
+At this point, the merchant can create and manage recommendation units from the Magento Admin UI then fetch those product recommendation units from their PWA storefront.
+
+## Install the Product Recommendations module
+
+Product Recommendations support on PWA requires installing the `venia-product-recommendations` package and the Product Recommendations Magento module.
+
+1. You can install the PWA `venia-product-recommendations` package from the NPM registry:
+
+   ```sh
+   npm install @magento/venia-product-recommendations
+   ```
+   This package contains everything you need to collect behavioral data and render the recommendations. Some recommendation types use behavioral data from your shoppers to train machine learning models that build personalized recommendations. Other recommendation types use catalog data only and do not use any behavioral data. See the [Magento user guide](https://docs.magento.com/user-guide/marketing/product-recommendations.html#trainmlmodels) to learn how Adobe Sensei trains machine learning models that results in higher quality recommendations.
+
+1. To install the Magento Product Recommendations backend module, see the [Magento developer documentation](https://devdocs.magento.com/recommendations/install-configure.html).
+
+1. Additionally, you need to install the `module-data-services-graphql` module that expands Magento's existing GraphQL API to include fields that are used to fetch the recommendations from the Recommendations Service.
+
+   ```sh
+   npm install @magento/module-data-services-graphql
+   ```
+
+## Create recommendation units
+
+Creating a product recommendation unit for your PWA storefront is similar to [creating one for a Magento storefront](https://docs.magento.com/user-guide/marketing/create-new-rec.html).
+The difference is that after you create a recommendation unit, you then need to add code to your PWA storefront to explicitly render the recommendation unit from the Magento backend.
+
+## Render recommendations
+
+To control the recommendation unit placement on your PWA storefront, use the local intercept file. See the [extensibility framework][] documentation to learn more. To render the recommendation unit on your PWA storefront, use one of the methods described below.
+
+### Venia UI component
+
+The `Recommendations` component is part of the [venia-ui package][].
+It contains React components that do the following:
+
+-  Collect and send behavioral data to Adobe Sensei
+-  Fetch recommendations from the recommendations service
+-  Render the recommendation unit to your storefront page
+
+#### Example
+
+```jsx
+//Render recommendations using visual component
+return <Recommendations pageType={CMS} />
```

Comment:
What does the import statement for this component look like?

Reply:
@abrhim can you take a look?

Reply:
import { VeniaProductRecommendations, PageTypes } from "@magento/venia-product-recommendations"

return <VeniaProductRecommendations pageType={PageTypes.CMS} />